### ISSUE TITLE
error An unexpected error occurred: "EINVAL: invalid argument, mkdir .../Library/Caches/Yarn/v1/npm-babel-plugin-syntax-async-functions-6.13.0-cad9cad1191b5ad634bf30ae0872391e0647be95'"

### ISSUE THREAD

**Issue by ngtlin** [yarnpkg/yarn#5645]
1.5.1

**Do you want to request a *feature* or report a *bug*?**

[yarn-error.log](https://github.com/yarnpkg/yarn/files/1898806/yarn-error.log)

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Please mention your node.js, yarn and operating system version.**

**Comment by arcanis** [comment]
You might not have the permission to remove the files inside .../Library/Caches/Yarn/v1/npm-react-native-maps-0.17.0-d10fa94a049432c46b6ee03fe9db84c83da65c8b/lib/ios/AirMaps.

Can you check that you own this folder and everything it contains? You can also try to remove it and run a new cache clean to unblock yourself.

**Comment by ngtlin** [comment]
Yes, I figured somewhat later. I do have permission to delete folder. Somehow,  the folder can't be removed even with 'sudo rm -rf  npm-react-native-maps-0.17.0-d10fa94a049432c46b6ee03fe9db84c83da65c8b/lib/ios/AirMaps'. Very strange problem on my Mac.
